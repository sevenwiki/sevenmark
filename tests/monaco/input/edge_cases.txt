A

Line with trailing spaces   
Empty line above

Mixed   whitespace	with	tabs

End without newline