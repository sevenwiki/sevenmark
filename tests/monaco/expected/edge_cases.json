[
  {
    "Text": {
      "content": "A",
      "location": {
        "end_column": 2,
        "end_line": 1,
        "start_column": 1,
        "start_line": 1
      }
    }
  },
  {
    "Text": {
      "content": "\n",
      "location": {
        "end_column": 1,
        "end_line": 2,
        "start_column": 2,
        "start_line": 1
      }
    }
  },
  {
    "Text": {
      "content": "\n",
      "location": {
        "end_column": 1,
        "end_line": 3,
        "start_column": 1,
        "start_line": 2
      }
    }
  },
  {
    "Text": {
      "content": "Line with trailing spaces   ",
      "location": {
        "end_column": 29,
        "end_line": 3,
        "start_column": 1,
        "start_line": 3
      }
    }
  },
  {
    "Text": {
      "content": "\n",
      "location": {
        "end_column": 1,
        "end_line": 4,
        "start_column": 29,
        "start_line": 3
      }
    }
  },
  {
    "Text": {
      "content": "Empty line above",
      "location": {
        "end_column": 17,
        "end_line": 4,
        "start_column": 1,
        "start_line": 4
      }
    }
  },
  {
    "Text": {
      "content": "\n",
      "location": {
        "end_column": 1,
        "end_line": 5,
        "start_column": 17,
        "start_line": 4
      }
    }
  },
  {
    "Text": {
      "content": "\n",
      "location": {
        "end_column": 1,
        "end_line": 6,
        "start_column": 1,
        "start_line": 5
      }
    }
  },
  {
    "Text": {
      "content": "Mixed   whitespace\twith\ttabs",
      "location": {
        "end_column": 29,
        "end_line": 6,
        "start_column": 1,
        "start_line": 6
      }
    }
  },
  {
    "Text": {
      "content": "\n",
      "location": {
        "end_column": 1,
        "end_line": 7,
        "start_column": 29,
        "start_line": 6
      }
    }
  },
  {
    "Text": {
      "content": "\n",
      "location": {
        "end_column": 1,
        "end_line": 8,
        "start_column": 1,
        "start_line": 7
      }
    }
  },
  {
    "Text": {
      "content": "End without newline",
      "location": {
        "end_column": 20,
        "end_line": 8,
        "start_column": 1,
        "start_line": 8
      }
    }
  }
]